src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandlerTest.java

```diff
@@ -28,7 +28,7 @@
 import static uk.gov.hmcts.reform.finrem.caseorchestration.TestSetUpUtils.newDocument;
 
 @RunWith(MockitoJUnitRunner.class)
-public class AmendApplicationAboutToSubmitHandlerTest extends BaseHandlerTest {
```

Comment:
Personal preference but I'd keep the name the way it was since it serves as a base test class

Reply:
renamed to BaseHandlerTestSetup - so it do not complain - about to add atleast one test